bin/install-rok8s-requirements

```diff
@@ -155,8 +155,8 @@ if ! hash vault 2>/dev/null; then
   echo Installing vault...
   cd "${ROK8S_INSTALL_PATH}"
   curl -LO "https://releases.hashicorp.com/vault/1.7.1/vault_1.7.1_linux_amd64.zip"
```

Comment:
This curl line will need to update as well

Reply:
Nice catch. Fixed